src/tests/functional/inference_engine/CMakeLists.txt

```diff
@@ -26,6 +28,8 @@ set(DEPENDENCIES
     template_extension
     openvino_template_extension
     lptNgraphFunctions
+    inference_engine_snippets
+    snippetsNgraphFunctions
```

Comment:
`DEPENDENCIES` should contain only dynamically loaded stuff, while common libraries are added as dependencies via `LINK_LIBRARIES` list

Reply:
Valid point, removed.